### ISSUE TITLE
progressive image loading

### ISSUE THREAD

**Issue by vipulasri** [Baseflow/flutter_cached_network_image#43]
Is there a way to have progressive image loading which is supported by JPEG and WEBp images?

**Comment by renefloor** [comment]
For the image supported progressive loading I will probably need a lot of rewriting as the loading and showing of the image is fully seperated. (File loading is even a seperate library.) Using two images is easier, as I can just start loading the second one when the first one is finished. I am not sure how much developers want something like this, as images for apps are often already optimized to be small.

**Comment by vipulasri** [comment]
I agree, then the library should offer an extra method/option like thumbnail() that would really help.

**Comment by renefloor** [comment]
I was thinking about a stream, that way you can change it to any amount of images/files. Do you know some easy to read documentation about this? Is this generic or does every image type has its own methods?

**Comment by vipulasri** [comment]
I can tell you about most commonly used Image Loading library on Android i.e [Glide](https://bumptech.github.io/glide/). It has great features as well as documentation

**Comment by perlatus** [comment]
Hey you can actually use the placeholder widget for this. That's what I'm doing. I pass in a placeholder widget that's basically a lower-res image (already cached) stacked with a progress bar.
